TrailheadApex11/ExercisesPage/src/classes/ExercisesLoader.cls

```diff
@@ -0,0 +1,7 @@
+/**
+ * Created by Kuba on 29.11.2017.
+ */
+
+public interface ExercisesLoader {
+    String solveTheExercise(List<String> input);
```

Comment:
Jakoś mi się to nie podoba. Ludzie z którymi współpracowałem dotychczas właściwie zawsze używają modyfikatora dostępu. Ja wiem, że tu jest public jako default bo to jest interface i private to się nawet nie da użyć, ale wolałbym go tu jednak po prostu widzieć. Jak możesz to wszędzie używaj modyfikatorów, mimo iż to teraz jest technicznie ok.